chucks-pick3-web/assets/css/navbar.css

```diff
@@ -146,9 +150,194 @@
 .sub-nav-bar-container .sub-nav-bar-col-50.allow-hover:hover {
     background-color: green;
     cursor: pointer;
+    text-decoration: none;
 }
 
 .sub-nav-bar-container .sub-nav-bar-col-50.active {
     background-color: gray;
 }
 
+/* Responsive Set Up------- */
+
+
+
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+.mobile-nav-bar-container {
+    display: none;
+}
+
+
+
+@media only screen and (max-width: 760px) {
+
+.main-nav-bar-container{
+    display: none;
+}
+    .sub-nav-bar-container {
+        display: none;
+    }
+
+    .sub-nav-bar-title-container {
+        display: none;
+    }
+
+    .float-right {
+        float: right;
+    }
+
+.mobile-nav-bar-container{
+    display: block;
+    width: 100%;
+    height: 50px;
+/*expands to 100% width */
+}
+
+    .mobile-nav-bar-container .nav-bar-col-25.allow-hover:hover {
+        background-color: green;
+        cursor: pointer;
+    }
+    .mobile-nav-bar-container a:hover {
+        text-decoration: none;
+        color: black;
+    }
+
+
+.logo-container {
+    width: 20%;
+    height: 75px;
+
+/* stays at 20% width, takes on height of header-img */
+}
+
+.header-img-container {
+    width: 80%;
+    height: 75px;
+/* stays at 80% width */
+}
+
+
+
+[class*="nav-bar-col-25"] {
```

Comment:
Maybe we should name this class nav-bar-col-20 instead of nav-bar-col-25 since we are putting the width to 20% instead of 25%.

Reply:
Changed names on class to nav-bar-col-20 on all pages.